styles/main/_content.scss

```
@@ -286,6 +286,80 @@
 			font-weight: bold;
 		}
 	}
+	// Description ---------------------------------------------------------- //
+	.album_description {
+		width: 100%;
+		margin: 10px 0 0 30px;
+		color: white;
+		font-size: 14px;
+		line-height: 19px;
+		h1,
+		h2,
+		h3 {
+			margin: 20px 0 20px;
+			&:first-child {
+				margin-top: 0;
+			}
+			&:last-child {
+				margin-bottom: 0;
+			}
+		}
+		p,
+		ol,
+		ul,
+		blockquote,
+		pre {
+			margin: 10px 0 10px;
+			&:first-child {
+				margin-top: 0;
+			}
+			&:last-child {
+				margin-bottom: 0;
+			}
+		}
+		h1 {
+			font-size: 24px;
+			line-height: 29px;
+			font-weight: bold;
+		}
+		h2 {
+			font-size: 20px;
+			line-height: 24px;
+			font-weight: bold;
+		}
+		h3 {
+			font-size: 16px;
+			font-weight: bold;
+		}
+		ol {
+			list-style: decimal inside;
+		}
+		ul {
+			list-style: disc inside;
```

```suggestion
			list-style: disc outside;
			padding-left: 40px;
```

See previous comment.